kfs-core/src/main/java/edu/arizona/kfs/sys/KFSKeyConstants.java

```diff
@@ -33,5 +33,8 @@
     public static final String ERROR_INCOME_TYPE_COA_REQUIRED = "error.income.type.coa.required";
     public static final String ERROR_INCOME_TYPE_CODE_REQUIRED = "error.income.type.code.required";
     public static final String ERROR_INCOME_TYPE_AMOUNT_REQUIRED = "error.income.type.amount.required";
-
+
+    // Procurement Cardholder
+    public static final String MESSAGE_BATCH_UPLOAD_TITLE_PCDH = "message.batchUpload.title.procurementCardHolder";

```

Comment:
This was the second merge conflict. Again, I didn't actually change anything except remove the strings that Git added denoting a merge conflict (<<<<<<< HEAD, =======, etc.)